modules/contracts-for-testing/src/main/kotlin/com/r3/corda/lib/tokens/testing/states/Rubles.kt

```diff
@@ -47,8 +47,8 @@ data class Appartment(val id: String = "Foo") : TokenType(id, 0)
 
 
 @BelongsToContract(DodgeTokenContract::class)
-open class DodgeToken(override val amount: Amount<IssuedTokenType<RUB>>,
-                      override val holder: AbstractParty) : FungibleToken<RUB>(amount, holder)
+open class DodgeToken(amount: Amount<IssuedTokenType>,
```

Comment:
This doesn't seem right, that DodgeToken theoretically should hold Rubles, but there is no Ruble in type

Reply:
There are no types any more! `DodgeToken` can now hold any TokenType.

Reply:
yeah, I know, that's why i am wondering why there is a DodgeToken in a first place :D

Reply:
I needed a token that had a different contract to test the condition that tokens cannot move to a type with a different contract